packages/ordma/ordma.0.0.1/opam

```diff
@@ -0,0 +1,19 @@
+opam-version: "1.2"
+maintainer: "Romain Slootmaekers <romain.slootmaekers@openvstorage.com>"
+authors: "Romain Slootmaekers <romain.slootmaekers@openvstorage.com>"
+homepage: "https://github.com/toolslive/ordma"
+bug-reports: "https://github.com/toolslive/ordma/issues"
+license: "TBD"
+dev-repo: "https://github.com/toolslive/ordma"
+build: [make lib]
+install: [make "install"]
+remove: ["make uninstall"]
+depends: [
+  "ocamlfind" {build}
+  "lwt" {>= "2.5.1"}
+]
+depexts: [
+  ["ubuntu"]
+  ["librdmacm-dev"]
+]
+available: [os != "darwin" | ocaml-version >= "4.02.0"]
```

Comment:
This is a surprising constraint.  Did you mean
```available: [os != "darwin" & ocaml-version >= "4.02.0"]```
?

If so, it'd be better to list the platforms where the package is available:
```available: [os = "linux" & ocaml-version >= "4.02.0"]```